src/libvirtApi/interface.js

```diff
@@ -24,52 +24,19 @@
 import store from '../store.js';
 
 import { updateOrAddInterface } from '../actions/store-actions.js';
-import { parseIfaceDumpxml } from '../libvirt-xml-parse.js';
-import { call, Enum, timeout } from './helpers.js';
 
-/*
- * Read properties of a single Interface
- *
- * @param {object} objPath interface object path
- * @param {string} connectionName
- */
-export async function interfaceGet({
-    id: objPath,
-    connectionName,
-}) {
-    const props = {};
+export async function interfaceGetAll() {
+    let ifaces = [];
 
     try {
-        const [resultProps] = await call(connectionName, objPath, 'org.freedesktop.DBus.Properties', 'GetAll',
-                                         ['org.libvirt.Interface'], { timeout, type: 's' });
-        /* Sometimes not all properties are returned; for example when some network got deleted while part
-            * of the properties got fetched from libvirt. Make sure that there is check before reading the attributes.
-            */
-        if ("Active" in resultProps)
-            props.active = resultProps.Active.v.v;
-        if ("MAC" in resultProps)
-            props.mac = resultProps.MAC.v.v;
-        if ("Name" in resultProps)
```

Comment:
I see these properties being used in other parts of the code. It shouldn't be too hard to get the MAC address and Active (aka. "UP") from the JSON output as well?

Reply:
Ah, must've missed them. But yes, they can be picked from the json output too

Reply:
Done in #1782.